contrib/pyln-client/pyln/client/plugin.py

```diff
@@ -84,7 +84,9 @@ def get_usage(self):
                 args.append("[%s]" % arg)
 
         if self.description is not None:
-            args.append("\n%s" % self.description)
+            doc = inspect.getdoc(self.func)
+            doc = re.sub('\n+', ' | ', doc)
+            args.append(" | %s" % doc)
 
         return " ".join(args)
 
```

Comment:
This should be in the next commit?

Also, the Changelog line for the first commit is unnecessary: this isn't a user-visible change!